bpf/lib/nodeport.h

```diff
@@ -2276,7 +2293,7 @@ static __always_inline int handle_nat_fwd(struct __ctx_buff *ctx)
 					       is_defined(IS_BPF_HOST)),
 					 is_defined(ENABLE_EGRESS_GATEWAY)),
 				   CILIUM_CALL_IPV4_ENCAP_NODEPORT_NAT,
-				   tail_handle_nat_fwd_ipv4);
+				   handle_nat_fwd_ipv4);
```

Comment:
Don't we usually prefer to use the prefix `tail_` for a tail called function,? Also I'm not sure to get the point I don't see other places where `tail_handle_nat_fwd_ipv{4,6}` is called, why do you keep it? I would say it's the whole purpose of your change I'm probably missing something...

Reply:
`invoke_tailcall_if()` is a conditional tail call. Here it either tail-calls to `CILIUM_CALL_IPV4_ENCAP_NODEPORT_NAT`, or inlines `handle_nat_fwd_ipv4()`.